### PR TITLE
Dynamic Bas domain fix

### PR DESCRIPTION
**Pull request by valerii-cognite**
That was a bug for Dynamic base domain story, a small fix to that

### CODE REVIEW

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-69.surge.sh

File: src/components/DataProvider/index.js [hunk 1]
```diff
@@ -121,6 +140,16 @@ export default class DataProvider extends Component {
       }
     });
 
+    if (
+      newDomain &&
```
Comment (evancharlton) [inline src/components/DataProvider/index.js]: Why not use `isEqual` here too?

Comment (valerii-cognite) [reply src/components/DataProvider/index.js]: That's a reasonable comment, fixed

**Comment by valerii-cognite** [comment]
> Is there a story to show this?

Yep, called 'Dynamic base domain'

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-69.surge.sh

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-69.surge.sh

**Comment by evancharlton** [comment]
And you have a typo in your commit message :-)

**Pull request closed without merging**